whirlpool/auth.py

```diff
@@ -95,8 +95,11 @@ async def _do_auth(self, refresh_token):
                         return json.loads(await r.text())
                     elif refresh_token:
                         return await self._do_auth(refresh_token=None)
+                    elif r.status != 200:
+                        raise aiohttp.ClientConnectionError
                     return None
-
+        except (aiohttp.ClientConnectionError, asyncio.TimeoutError) as ex:
+            raise ConnectionError("Cannot connect")
```

Comment:
```suggestion
        except asyncio.TimeoutError as ex:
            raise ex
```
Could we simply re-raise the timeout? Errors that are not timeouts should not be treated as such.

Reply:
I think we should keep the ClientConnectionError and TimeoutErrors.... see proposed